Automattic/woocommerce-payments
# Currency switcher shouldn't affect Order confirmation screen prices

Fixes #2486
#### Changes proposed in this Pull Request

Tweak FrontendCurrencies to override the currency to order currency on specific pages:
- Checkout -> Order Received
- My Account -> View Orders.

Currency switching does not affect the price formatting of the order. See the screenshots below.

![image](https://user-images.githubusercontent.com/79862886/132864844-3850ac24-9db5-4a70-84b6-d02ddc028f62.png)

![image](https://user-images.githubusercontent.com/79862886/132864810-3d805621-5014-4219-9a93-0ccedd832778.png)

#### Testing instructions

* Ensure you have three currencies: EUR, USD, and JPY.
* Go to the store, switch the currency to Euro and buy some product.
* On the order received page, try switching the currency to USD and JPY. Order price format (short and explicit) should not change.
* Go to My account -> Orders. Select the order and click to go to the view order page.
* On the view order page, try switching the currency to USD and JPY. Price format (short and explicit) should not change at the order details section.

-------------------


**Post merge**
